docs/maintenance/pr-community-board.md

```diff
@@ -0,0 +1,41 @@
+# PR Community board workflow and guidelines
+
+**Intended audience**
+
+This guide is intended for contributors, coordinators and developers of InvenioRDM.
+
+**Scope**
+
+The guide covers the recommended way of coordinating community pull requests integration within the InvenioRDM. The goal of the project is to respond to the commmunity need of facilitating the review and integration processes of the **pull requests which are not the part of the main sprint**.
+
+## Board
+
+You'll find the latest board [here](https://github.com/orgs/inveniosoftware/projects/109/views/1).
+
+
+## Roles
+
+- **Coordinator**: Person who coordinates the integration process, has the overview of the current pull requests and is in charge of the communication between contributors, reviewers, maintainers, and release manager.
+- **Contributor/submitter**: person who contributes by creating pull requests.
+- **Reviewer**: Person who is contributing by reviewing contributors' work. 
+- **Maintainer**: Person who has merging rights in the InvenioRDM repositories, responsible for integrating the pull requests into the product.
+- **Release manager**: Person in charge of the current main InvenioAppRDM release.
+
+## Columns
+
+- **Triage**: **Submitters** place the PRs in this column. PR is ready to be prioritised by the **coordinator**. 
+- **Work in progress**: Changes/suggestions were requested by the reviewers, the submitter is addressing them.
+- **To review**: PR is ready to be reviewed by the reviewers.
+- **Discussion**:  PR needs to be re-discussed in terms of solutions - fe. there are different opinions on the implementation.
+- **Approved**: PR was approved, two reviewers agree on integrating it. Ready to be merged.
+- **To release**: PR is ready to be released. **Maintainers** or **release manager** decide on when it should happen. PRs might be merged or not merged yet.
+- **Released/done**: Archive of recently integrated PRs - useful for tracking the latest changes
```

Comment:
One question that came up with @fenekku that we haven't discussed is how long PR's will stay in the done column. What does "latest" mean in terms of time?

Reply:
the one since the last major invenio-app-rdm release. Release manager could be the one responsible to archive the column when the major is released